OKTA-804418 Deprecate Apps API: update /apps API links to Redocly (#5092)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

Retrieves a list of applications mapped to a policy

> > **Note:** To retrieve a list of applications mapped to a policy, use [List all resources mapped to a Policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/#tag/Policy/operation/listPolicyMappings). If you need to assign an application to a specific policy, use the [Assign an authentication policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationPolicies/#tag/ApplicationPolicies/operation/assignApplicationPolicy) operation of the Apps API.

Reviewer: ```suggestion
> **Note:** To retrieve a list of apps mapped to a policy, use [List all resources mapped to a policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/#tag/Policy/operation/listPolicyMappings). If you need to assign an app to a specific policy, use the [Assign an authentication policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationPolicies/#tag/ApplicationPolicies/operation/assignApplicationPolicy) operation of the Apps API.
```

> > **Note:** To assign an application to a specific policy, use the [Update application policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationPolicies/#tag/ApplicationPolicies/operation/assignApplicationPolicy) operation of the Apps API.

Reviewer: ```suggestion
> **Note:** To assign an app to a specific policy, use the [Update app policy](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationPolicies/#tag/ApplicationPolicies/operation/assignApplicationPolicy) operation of the Apps API.
```

##### Response types

HTTP 200:

> Array of [Application objects](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/listApplications)

Reviewer: ```suggestion
Array of [app objects](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/listApplications)
```
